omnia_timeseries_sdk/client.py

```diff
@@ -47,28 +51,36 @@ def __init__(self, base_url: str = None, log_level: str = None):
     @staticmethod
     def _token_request():
         """Requests access token."""
-        resource = os.getenv("omniaResourceId")
-        client_id = os.getenv("omniaClientId")
+        # Set resource id and client id. Default ids for authentication by user impersonation (without shared secret).
+        # service-to-service / machine-to-machine authentication using a shared secret requires generally different
+        # resource and client ids and a shared secret (client secret). See also
+        # https://github.com/equinor/OmniaPlant/wiki/Authentication-&-Authorization
+        resource_id = os.getenv("omniaResourceId", "141369bd-3dca-4b55-825b-56ad4a69b1fc")
```

Comment:
Are you sure these can be online? Same holds for the tenant ID. I have heard conflicting information about wether it is OK or not.

Reply:
First of all, thanks you for taking the time reviewing and proposing changes.

When I started this project I asked around and never really got a clear answer.  Considering that resource and client IDs are publicly available on https://github.com/equinor/OmniaPlant/wiki/Authentication-&-Authorization and the fact that it is useless without a shared secret or an equinor user with the necessary privileges, I decided to move on with these defaults.

Do you agree?

Reply:
Should probably be ok then. I also found that we have exposed the tenant ID in public github issues, so I hope its fine :-p